index.js

```diff
@@ -91,6 +99,39 @@ class Store {
     return this;
   }
 
+  /**
+   * Assign `value` to `key` while retaining prior members of `value` if
+   * `value` is a map. If `value` is not a map, overwrites like `.set`.
+   *
+   * ```js
+   * store.set('a', { b: c });
+   * //=> {a: { b: c }}
+   *
+   * store.merge('a', { d: e });
+   * //=> {a: { b: c, d: e }}
+   *
+   * store.set('a', 'b');
+   * //=> {a: 'b'}
+   *
+   * store.merge('a', { c : 'd' });
+   * //=> {a: { c : 'd' }}
+   * ```
+   *
+   * @name .merge
+   * @param {String} `key`
+   * @param {any} `val` The value to merge to `key`. Must be a valid JSON type: String, Number, Array or Object.
+   * @return {Object} `Store` for chaining
+   * @api public
+   */
+
+  merge(key, val) {
+    let oldVal = this.get(key);
+    if (oldVal && typeof oldVal == "object" && ! Array.isArray(oldVal)) {
```

Comment:
Please use single quotes, strict equals and remove the space before `Array.isArray`:

```suggestion
    if (oldVal && typeof oldVal === 'object' && !Array.isArray(oldVal)) {
```